2021/23xxx/CVE-2021-23872.json

```diff
@@ -68,6 +68,12 @@
                         "value": "CWE-269: Improper Privilege Management"
```

Comment:
("Expecting ',' delimiter: line 71 column 17 (char 2397)",)

Reply:
Updated formatting in second commmit